src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -547,16 +547,21 @@ namespace OpenMS
     ProteaseDigestion pd;
     pd.setEnzyme("Trypsin");
 
-    size_t feature_idx = 0;
+    auto& mrmfeatures = transition_group_detection.getFeaturesMuteable();
+
     // Go through all peak groups (found MRM features) and score them
-    for (std::vector<MRMFeature>::iterator mrmfeature = transition_group_detection.getFeaturesMuteable().begin();
-         mrmfeature != transition_group_detection.getFeaturesMuteable().end(); ++mrmfeature)
+    #ifdef _OPENMP
+    int in_parallel = omp_in_parallel();
+    #endif
+    #pragma omp parallel for if(in_parallel == 0) schedule(dynamic,1)
+    for (Size feature_idx = 0; feature_idx < mrmfeatures.size(); ++feature_idx)
```

Comment:
this shouldn't work. OpenMP 2.0 only allows int as loop variable

Reply:
I think it is all integral types, not integer types: https://stackoverflow.com/questions/14033280/openmp-index-variable-in-openmp-for-statement-must-have-signed-integral-type#:~:text=An%20integral%20type%20is%20one,index%20of%20your%20for%20loop
We will see.

Reply:
Size is not signed

Reply:
could be. Maybe it was size_t that didn't work (because it is unsigned)

Reply:
you need `SignedSize` or `int` etc for VS .. unfortunately

Reply:
very unfortunate